README.md

```diff
@@ -1,5 +1,10 @@
 # QCVIS — Interactive Visualization of Quantum Algorithms
 
+[![License](https://img.shields.io/github/license/fh-igd-iet/qcvis?color=blue&label=License)](https://fh-igd-iet.github.io/qcvis/)
+[![Try it out](https://img.shields.io/badge/-Try%20it%20out%20→-brightgreen "Open application")](https://fh-igd-iet.github.io/qcvis/)
```

Comment:
- What is the advantage of giving this badge a title (` "Open application"` in the parentheses) when alt-text is already given by the initial `[Try it out]`?

Reply:
My concern was that "Try it out" could be too unclear - hence the additional title. But you're probably right, I admit that the chance that somebody who's curious hovers the button instead of just clicking it is rather small.

Reply:
Thanks, I was mainly thinking of people using a screen-reader or other accessibility tools who generally get read the alt-text not the title by default as far as I know